.gitignore

```diff
@@ -12,3 +12,6 @@ firmware.code-workspace
 
 # Clion
 *.idea/
+
+# hw_bridge repo
+uwrt-mars-rover-hw-bridge
```

Comment:
why did you add the submodule to gitignore? Don't we want to track submodule updates?

Reply:
by nature of git the contents will be ignored and just the submodule reference is tracked -- at least in all of my experience with submodules